### PR TITLE
5008 maas automated hardware testing

### LINKED ISSUE #5008: [Takeover + landing] MAAS Automated Hardware Testing WP - Build

### PR DESCRIPTION
**Pull request by solazio**
## Done

Add maas takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/maas-hardware-testing](http://0.0.0.0:8001/engage/maas-hardware-testing) and  [http://0.0.0.0:8001/](http://0.0.0.0:8001/) 
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Refresh home page until maas takeover comes up.
- Make sure it complies with the [brief](https://docs.google.com/document/d/1xyNrWYsYcvux_gYtoWw39XD9cP4QmiAYJnDlLuV8CvE/edit?ts=5cab9e6f)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5006)

## Issue / Card

Fixes #5008

### CODE REVIEW

File: templates/takeovers/_maas-hardware-testing_takeover.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          href="/engage/maas-hardware-testing"
+          class="p-button--positive"
+          >Download the whitepaper
+        </a>
+      </p>
+    </div>
+  </div>
+  <style>
+    .p-takeover {
+    background-image: url("{{ ASSET_SERVER_URL }}127c8ba7-hero-bg-maas-2.svg"), linear-gradient(180deg, #222222 0%, #111111 100%);
```
Comment (deadlight) [inline templates/takeovers/_maas-hardware-testing_takeover.html]: fix indentation here

**Comment by anasereijo** [comment]
@solazio looking good! just 2 small things:

Takeover:
- [ ]  title doesn't match what we have on the brief

Landing page:
- [ ] change the `h4` to `p` as it's looking weird with so much text

Thank you

**Comment by anasereijo** [comment]
@solazio at the moment it is linking to the wrong pdf. @SirSamTumless can you please ask marketing to update the [brief](https://docs.google.com/document/d/1SfZKopMr0ZXnxT5h1cEEebCysp6Carf9GSPzkL51c8E/edit#)?

**Comment by pmahnke** [comment]
@solazio the thank-you page is for IOT and I assume the form ID is wrong too.

we need to check this too I guess.  I left a comment for Katie and Lucia on the breif

**Comment by pmahnke** [comment]
@anasereijo and @SirSamTumless I added a comment on the brief... I didn't see your comment... also, please block or -1 something if you see an issue

**Comment by solazio** [comment]
@pmahnke I am aware I need to update the form ID and the thank-you page once I get that info.

File: templates/takeovers/_maas-hardware-testing_takeover.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  <div class="row u-clearfix u-vertically-center">
+    <div class="col-5">
+      <h1>
+        Build the data centre of the future 
+     </h1>
+      <p class="p-heading--four u-sv2">
+        Learn how to provision and manage bare metal servers like a cloud with MAAS.
+      </p>
+      <a
+        href="/engage/maas-hardware-testing"
```
Comment (caldav) [inline templates/takeovers/_maas-hardware-testing_takeover.html]:
```suggestion
        href="/engage/maas-hardware-testing?utm_source=takeover&utm_campaign=CY19_DC_MAAS_Whitepaper_HDWTest"
```

File: templates/takeovers/_maas-hardware-testing_takeover.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="col-7">
+      <p class="p-takeover__image u-align-text--center">
+        <img
+          src="{{ ASSET_SERVER_URL }}67e3e0f1-MAAS_white-orange_rgb.svg"
+          alt=""
+        />
+      </p>
+      <p class="u-hide--medium u-hide--large">
+        <a
+          href="/engage/maas-hardware-testing"
```
Comment (caldav) [inline templates/takeovers/_maas-hardware-testing_takeover.html]:
```suggestion
          href="/engage/maas-hardware-testing?utm_source=takeover&utm_campaign=CY19_DC_MAAS_Whitepaper_HDWTest"
```

**Comment by caldav** [comment]
(waiting on form stuff to re-review status)

**Comment by anasereijo** [comment]
@caldav do you know if the whitepaper still in progress?

**Comment by SirSamTumless** [comment]
@anasereijo @caldav 

The whitepaper itself (PDF) still needs a final review from Sarah.

**Comment by caldav** [comment]
Since this may be one of the next engage page to land, we should use the [new metadata format](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5261) (@solazio) and add a meta_image (@anasereijo).

**Comment by anasereijo** [comment]
> Since this may be one of the next engage page to land, we should use the [new metadata format](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5261) (@solazio) and add a meta_image (@anasereijo).

@SirSamTumless should we create a brief for this or an issue for next iteration so we don't forget to do this?

**Comment by SirSamTumless** [comment]
> > Since this may be one of the next engage page to land, we should use the [new metadata format](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5261) (@solazio) and add a meta_image (@anasereijo).
> 
> @SirSamTumless should we create a brief for this or an issue for next iteration so we don't forget to do this?

@caldav Do you think we should do this for the last few? If that's the case, I will make a larger epic.

**Comment by caldav** [comment]
@SirSamTumless defintely, I mention 5 to 10 in the bug report, but here is a [list sorted by traffic, hence the most likely to be shared socially](https://analytics.google.com/analytics/web/#/report/content-pages/a1018242w108544136p113106543/_u.dateOption=last30days&explorer-table.filter=www.ubuntu.com~2Fengage&explorer-table.plotKeys=%5B%5D&explorer-table.rowCount=25&_.useg=). I believe if we do all new ones as well as attacking this list from the top, we should be fine.

**Comment by pmahnke** [comment]
@solazio we also need to add all these to the new /engage index page... don't know if we missed any before.

**Pull request closed without merging**